client/scanner.js

```diff
@@ -83,7 +84,7 @@ class Scanner {
 
   isValidBlock(block) {
     if (!block) {
-      this.log.error("")
+      this.log.error('')
```

Comment:
Shouldn't we have an error message if we're logging something? `Invalid block` would suffice. Or, we can take out the logging entirely if this is an internal function that just needs the return result.